controller/login.go

```diff
@@ -115,7 +122,14 @@ func (c *LoginController) Generate(ctx *app.GenerateLoginContext) error {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, errs.Wrap(err, "unable to generate test token")))
 	}
 	// Creates the testuser2 user and identity if they don't yet exist
-	c.auth.CreateOrUpdateKeycloakUser(*testuser.Token.AccessToken, ctx)
+	_, _, err = c.auth.CreateOrUpdateKeycloakUser(*testuser.Token.AccessToken, ctx)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"err":      err,
+			"username": c.configuration.GetKeycloakTestUserName(),
```

Comment:
c.configuration.GetKeycloakTestUser2Name()

Reply:
Hope this fixes it https://github.com/kishansagathiya/fabric8-wit/commit/0f1ea8ffcb7aa249478b5a8ac51c22d978c19d8c